kata-deploy/scripts/kata-deploy.sh

```diff
@@ -151,7 +151,11 @@ function configure_containerd_runtime() {
 		runtime+="-$1"
 		configuration+="-$1"
 	fi
-	local runtime_table="plugins.cri.containerd.runtimes.$runtime"
+	local pluginid=cri
+	if grep -q "version = 2" $containerd_conf_file; then
```

Comment:
Thanks @dsyer. Please could you tweak slightly to anchor the regex and quote the conf file:

```sh
if egrep -q "version = 2\>" "$containerd_conf_file"; then
```